covid_moonshot_ml/datasets/moonshot.py

```diff
@@ -1,6 +1,6 @@
 BASE_URL = "https://app.collaborativedrug.com/api/v1/vaults/5549/"
 ## All molecules with SMILES (public)
-ALL_SMI_SEARCH = "searches/8975987-kmJ-vR0fhkdccPw5UdWiIA"
+ALL_SMI_SEARCH = "searches/9469227-zd2doWwzJ63bZYaI_vkjXg"
```

Comment:
Why did the saved search change?

Reply:
I made a new saved search on CDD that contains the upper/lower CI in the export, so updated to the URL for the new search